Feature/incrwmean (#256)
stdlib-js/stdlib: lib/node_modules/@stdlib/stats/incr/wmean/docs/repl.txt

{{alias}}()
    Returns an accumulator function which incrementally computes a weighted
    arithmetic mean.

>     If provided parameter(s), the accumulator function returns an updated

Reviewer: ```suggestion
    If provided arguments, the accumulator function returns an updated
```

>     weighted mean. If not provided a value, the accumulator function returns

Reviewer: ```suggestion
    weighted mean. If not provided arguments, the accumulator function returns
```

    the current weighted mean.

    If provided `NaN` or a value which, when used in computations, results in
    `NaN`, the accumulated value is `NaN` for all future invocations.

>     The accumulator accepts two parameters, the first is the value and the

Reviewer: ```suggestion
    The accumulator accepts two arguments: the first is the value and the
```
